app/scripts/lieferplanungen/lieferplanung.model.js

```diff
@@ -75,6 +75,11 @@ angular.module('openolitor-admin')
         isArray: true,
         url: API_URL + 'lieferplanungen/:id/lieferungen/:lieferungId/:korbStatus/aboIds'
       },
+      'getZusatzaboIdsByKorbStatus': {
+        method: 'GET',
+        isArray: true,
+        url: API_URL + 'lieferplanungen/:id/lieferungen/:lieferungId/:korbStatus/zusatzaboIds'
```

Comment:
What you actually get are "HauptaboIds", no? So the route could be "lieferplanungen/:id/lieferungen/:lieferungId/:korbStatus/hauptaboIds"

Reply:
you are right! I'll change it